backend/game/player/player.py

```diff
@@ -9,6 +9,7 @@ def __init__(self):
         self.is_alive = True
         self.victory_points = constants.DEATH_HIT_POINT
         self.energy = constants.DEFAULT_ENERGY_CUBE
+        self.card_on_hand = {}
```

Comment:
As stated in a previous review.... due to duplicate cards, a dictionary is not a valid way to store them